gui/actionables/actionables.md

```diff
@@ -0,0 +1,108 @@
+
+On "Text" only triggers, the user must first highlight a block of text in order to parse the trigger. Using "Text" mode triggers can be useful where the text that should be triggered on cannot be meaningfully matched by a regular expression, such as a number. In this case, a "Click and Text" mode trigger would hyperlink all numbers.
+
+![](actionables-trigger2.png)
+
+### Grouping Actionables
+
+By default, actionables menus are named by the name of the actionable itself. Actionables with the same name are automatically grouped by menu. Additionally, you can provide the menu name of an actionable to force it into another menu. This is helpful to group like actionables.
+
+For example, the CoreDNS Kit provides several actionables based on the format of the domain names. Each of these actionables are put in the "DNS" menu, as shown below.
+
+![](actionables-grouping.png)
+![](actionables-grouping2.png)
+
+## Actions
+
+Actions provide operations that can be executed on text matched by an actionable trigger. An actionable can contain any number of actions. Actions include opening URLs, launching other searches, and more.
+
+### The `_VALUE_` Variable
+
+Some actions allow using the text of a capture group from the regular expression of a trigger to be used in the action itself. For example, we can use the contents of the "color" capture group in a URL:
+
+```The color is (?<color>.*)```
+
+The capture group contents can then be used in a URL, using `_VALUE_` for the matched text:
+
+```https://en.wikipedia.org/wiki/_VALUE_```
+
+The keyword `_VALUE_` is the default placeholder for the matched text, and can be changed in some actions.
+
+### Action Types
+
+Gravwell provides several actions, and an actionable can use any or all actions in a single actionable. 
+
+#### Run a Query
+
+This action simply runs a new query, with the contents of the matched text as `_VALUE_`.
+
+#### Execute a Template
+
+The template action runs a pre-made template, using the matched text as the input variable to the template. 
+
+#### Launch a Dashboard
+
+The launch dashboard action opens a dashboard. If the dashboard has template variables, the user is prompted to select which variable to populate with the matched text.
+
+![](actionables-dashboard.png)
+
+#### Open a URL
+
+The URL action supports opening a new window/tab with a given URL and matched text, and additionally provides a set of timestamp options for providing the time range arguments from the search that the actionable triggered on. 
+
+The "Open in a modal" option opens the URL in a window within the current Gravwell instance, similar to an HTML iframe.
+
+#### Run a Saved Query
+
+Simply run a query from the Query Library. 
+
+
```

Comment:
Add a section about names and grouping in that menu?  Maybe show one with overlapping names in the menu vs. not?